docker-compose.yml

```diff
@@ -14,6 +16,9 @@ services:
       - COINRANKING_URL=http://wiremock:8080
       - COINRANKING_TOKEN=suchtoken
       - UNIVERSE_JWT_SECRET=suchsecret
+      - PROM_ADDRESS=https://testnet2-prometheus.mysterium.network/vm/
+      - PROM_USER=promclient
+      - PROM_PASS=eFM69rk(LCDh@rlGJoB&s_6*3Q
```

Comment:
Is this an actual password? Because the repo is public. Or we not care?

Reply:
rotated...